libcxx/test/std/numerics/simd/simd.class/simd.ctor/broadcast.pass.cpp

```diff
@@ -0,0 +1,91 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <simd>
+
+// REQUIRES: std-at-least-c++26
+
+#include <cassert>
+#include <variant>
+#include <simd>
+#include <type_traits>
+
+#include "type_algorithms.h"
+#include "../../utils.h"
+
+namespace dp = std::datapar;
```

Comment:
You seem to be missing tests for `noexcept`-ness as well. This probably applies elsewhere?